rpc_jobs/rpc_jenkins_etl.yml

```diff
@@ -26,3 +26,29 @@
     jobs:
       - 'Pull-Request-Whisperer_{repo}'
 
+- project:
+    name: "rpc-jenkins-etl-dev-premerge"
+    repo_name: "rpc-jenkins-etl"
+    repo_url: "https://github.com/rcbops/rpc-jenkins-etl"
```

Comment:
This is a private repo, so git@github.com:rcbops/rpc-jenkins-etl.git must be used so that jenkins can authenticate.